services/speech_to_text/v1.js

```diff
@@ -85,20 +98,39 @@ module.exports = function (RED) {
 
       var s2t = function (audio, format, cb) {
         if (format !== 'wav' && format !== 'flac' && format !== 'ogg') {
-          var message = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
-          node.error(message, msg)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2399943310/issues/source?bid=3225115&fileBranchId=3231190#l101)